### PR TITLE
Release Xtensor.jl 0.0.2

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: Xtensor/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+BinDeps 0.4.2
+Compat 0.21
```
Comment (tkelman) [inline Xtensor/versions/0.0.2/requires]: you're still not using Compat anywhere

Comment (SylvainCorlay) [inline Xtensor/versions/0.0.2/requires]: Fixed. Thanks for the review.

Comment (tkelman) [inline Xtensor/versions/0.0.2/requires]: be sure to make the corresponding change at https://github.com/QuantStack/Xtensor.jl/blob/master/REQUIRE before the next tag

Comment (SylvainCorlay) [inline Xtensor/versions/0.0.2/requires]: Done

**Pull request merged**